Library/Formula/puddletag.rb

```diff
@@ -55,6 +49,9 @@ def install
 
     bin.install Dir[libexec/"bin/*"]
     bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    system "sh create_macos_app_bundle.sh --name Puddletag --icon puddletag.png --script #{bin}/puddletag"
```

Comment:
Yeh, I'm :ok: with this case being included (and default) given what I said in the past and `linkapps` existing. Hopefully we can :skull: that command and `.app` bundles altogether.